deno/common/ts_morph_common.d.ts

```diff
@@ -345,28 +345,41 @@ export interface TsSourceFileContainer {
 /** Decorator for memoizing the result of a method or get accessor. */
 export declare function Memoize(target: any, propertyName: string, descriptor: TypedPropertyDescriptor<any>): void;
 
+/** Minimal attributes to show a error message with source */
+export declare type MaybeTraceAbleNode = undefined | null | {} | {
```

Comment:
Maybe just undefined?

```suggestion
export declare type MaybeTraceAbleNode = undefined | {
```

Reply:
Done 👍

Is there any way to use `Node` type from `ts-morph` inside the `common` package?